pkg/opaqueports/defaults.go

```diff
@@ -0,0 +1,17 @@
+package opaqueports
+
+// DefaultOpaquePorts is the default list of opaque ports that the destination
+// server will use to determine whether a destination is an opaque protocol.
+// When a pod or service already has its own annotation, that value will have
+// priority of this.
+//
+// Note: Keep in sync with proxy.opaquePorts in values.yaml
+var DefaultOpaquePorts = make(map[uint32]struct{})
```

Comment:
TIOLI: might this be more idiomatic as:

```
var DefaultOpaquePorts = map[uint32]struct{}{
  25: struct{}{},
  443: struct{}{},
  ...etc...
}
```

I think it's probably equivalent to what you have here, but I have a primal fear of the init function.

Reply:
Ah yep I meant to remove this thanks for catching. I had in my head when starting this that I needed to jump through these hoops to initialize the map.